.travis.yml

```diff
@@ -13,6 +13,7 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then if [[ ! $(which $SH) ]] ; then sudo apt-get -qq update        ; fi ; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then if [[ ! $(which $SH) ]] ; then sudo apt-get install -y $SH    ; fi ; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then if [[ ! $(which nc) ]]  ; then sudo apt-get install -y netcat ; fi ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then if [[ ! $(which strings) ]]  ; then sudo apt-get install -y binutils ; fi ; fi # for `strings` command
```

Comment:
`strings` command is used line 26 and 27 in `test.sh`.
This is used to generate random id/password.
Even though it is POSIX command, some of the OS does not have it (I tried it on the Ubuntu docker container, but it does not have the command...)

```bash
    export TTCP_ID="$(cat /dev/urandom | strings | grep -o '[[:alnum:]]' | tr -d '\n' | fold -w 128 | head -n 1)"
    export TTCP_PASSWORD="$(cat /dev/urandom | strings | grep -o '[[:alnum:]]' | tr -d '\n' | fold -w 128 | head -n 1)"
```

Anyway, I noticed that this part can be done without `strings` command like this.

```bash
cat /dev/urandom | grep -ao '[a-zA-Z0-9]' | tr -d '\n' | fold -w 128 | head -n 1
```

Therefore, I will delete the part you pointed out lol.